### ISSUE TITLE
Menu component does not read aria-label when accessibility is menuBehavior

### ISSUE THREAD

**Issue by tyhilde** [microsoft/fluentui#16073]
### Environment Information

- **Package version(s)**: 0.51.3
- **Browser and OS versions**: Chrome and Edge | Windows 10

### Describe the issue:

When you put an aria-label onto a Menu component,  Narrator does not read out the aria-label on the content, it goes straight to reading the first menu item. 

This issue only seems to happen on Menu with the default `accessibility={menuBehavior}`. If you put ` accessibility={menuAsToolbarBehavior}`, then the aria-label is read correctly.

### Please provide a reproduction of the issue in a codepen:

With default menuBehavior (does not work as expected):
https://codesandbox.io/s/pqrtl?module=/example.js
Note: The aria-label does show in the DOM
![image](https://user-images.githubusercontent.com/18689469/100272956-d71e4700-2f10-11eb-9822-2e5a14d22124.png)

With menuAsToolbarBehavior (works as expected):
https://codesandbox.io/s/1gwob?module=/example.js
![image](https://user-images.githubusercontent.com/18689469/100273043-00d76e00-2f11-11eb-8dcd-c76cb63d20ec.png)

#### Actual behavior:

Using Narrator, tab into the menu and it reads:  "Editorials, Menu Item, 1 of 3

#### Expected behavior:

Using Narrator, tab into the menu and it should read:  "Test Label, Editorials, Menu Item, 1 of 3

Using the example w/ menuAsToolbarBehavior you will see that Narrator reads the aria-label first.

**Comment by jurokapsiar** [comment]
Ut is quite unusual to give a label to menu as the menu is not supposed to be present in the page for a long time - it is expected to be used in a menu button mostly. For toolbar, it is different, as toolbars are always present in the page. Actually, if there are multiple toolbars, label is mandatory. This is expected screen reader behavior.
